config/routes.rb

```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
-  resources :books, only: [:show]
+  resources :books, only: [:show] do
+    resources :reviews
+  end
```

Comment:
Make sure to read your user stories carefully. They all ask for a book show so you don't need any of these other routes.